homeassistant/components/switch/rpi_gpio.py

```diff
@@ -83,13 +96,13 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """ Turn the device on. """
-        if self._switch(True):
+        if self._switch(True if self._active_state == "HIGH" else False):
```

Comment:
I'm not sure if I understand the `_active_state` check here. Active state is a Config variable, basically if not set to default value, it will invert the call to `_switch`? If so, you can do only 1 check in the `__init__:
```python
self._active_state = active_state == "HIGH"
```
And rewrite `turn_on` to be:
```python
if self._switch(self._active_state):
```
And rewrite `turn_off` to be:
```python
if self._switch(!self._active_state):
```

But since it is hardcoded to check versus high anyway, maybe we should change the config option to be called invert?